react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -331,6 +332,18 @@ class Filmstrip extends PureComponent <Props, State> {
                         </div>
                     </div>
                 )}
+                {_localScreenShare && !_verticalViewGrid && (
```

Comment:
do we also want to hide the local screen share tile when `_disableSelfView` is true?

Reply:
I think so, it's another form of "self view".

Reply:
Yes all local preview thumbnails need to be hidden when self view is disabled.